CRM/Twingle/Form/Profile.php

```diff
@@ -14,6 +14,8 @@
 +-------------------------------------------------------------*/
 
 use CRM_Twingle_ExtensionUtil as E;
+use CRM_Twingle_Exceptions_ProfileException as ProfileException;
+use CRM_Twingle_Exceptions_ProfileValidationError as ProfileValidationError;
```

Comment:
Using the `Civi` namespace would avoid long class names, see above comment.

Reply:
Done!